ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ip-security.md

```diff
@@ -0,0 +1,89 @@
+# ip-security
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+  - [IP Security](#ip-security)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+### IP Security
+
+##### IPSec profiles
+
+| Profile name | IKE policy | SA policy | Connection | DPD Interval | DPD Time | DPD action | Mode |
+| ------------ | ---------- | ----------| ---------- | ------------ | -------- | ---------- | ---- |
+| Profile-1 | IKE-1 | SA-1 | start | - | - | - | transport |
+| Profile-2 | - | SA-2 | start | - | - | - | tunnel |
+
+##### Key controller
+
+| Profile name |
+| ------------ |
+| Profile-1 |
+
+#### IP Security Configuration
+
+```eos
+!
+ip security
+   ike policy IKE-1
+      local_id 192.168.100.1
+   !
+   ike policy IKE-2
```

Comment:
Should we add a table for ike policy or maybe just add a parentheses after the IKE policy like `IKE-1 (ID 192.168.100.1)` ?

Reply:
typo in template ... :) fixing it!